WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1564,36 +1553,32 @@ private void trackEditorCreatedPost(String action, Intent intent) {
         );
     }
 
-    private boolean updatePostObject(boolean isAutosave) {
-        if (!mEditPostRepository.hasPost() || mEditorFragment == null) {
-            AppLog.e(AppLog.T.POSTS, "Attempted to save an invalid Post.");
+    private boolean updatePostObject() {
+        if (mEditorFragment == null) {
+            AppLog.e(AppLog.T.POSTS, "Fragment not initialized");
+            return false;
+        }
+        try {
+            UpdateResult updateResult = mViewModel
+                    .updatePostObject(this, mShowAztecEditor, mEditPostRepository, (String) mEditorFragment.getTitle(),
```

Comment:
The original idea was that `getContent` needs a field from the `PostModel` and that's only read in a transaction. However, I'm changing this logic so it will be both loaded in the lambda.

Reply:
👍 Thanks!